deployment/ansible/deploy.yml

```diff
@@ -65,6 +65,13 @@
       when: local_package is defined
       tags: [package]
 
+    - name: "Make sure {{ db_path }} exists"
```

Comment:
There is no such variable.
Also if it's set to "." it will read: `"Make sure . exists"` which looks ugly to me.
That's why I suggested hardcoded term instead of var.

Reply:
I just tested it and it reads "Make sure {{ db_path }} exists" so it's working anyway.
Change only if you want.